privacy-statement.md

```diff
@@ -0,0 +1,205 @@
+request identifier or your Wikipedia username. Account Creators who violate this policy are terminated from the Project
+and may face additional sanction, as prescribed by the Wikimedia Foundation.
+
+**Requesters should be aware of the following important information:**
+1. Your real name is considered private information, **unless you include your real name in your requested username**.
+   If you do not want your real name to be associated with your Wikipedia account, **do not include it in your requested
+   username**. If you do choose to include your real name in your username, it will be permanently tied to your
+   request identifier in the Tool, even if you later change your username, and will be visible to the entire public on
+   Wikipedia itself, potentially forever. For more information, please see [Wikipedia's policy on the use of real names
+   as usernames][enwiki-upol-real-names].
+2. **Do not include any password as part of your account request, or in any email communication with the Account Creator
+   team**. A random password will be automatically emailed to you if your account is created, and you will be able to
+   change it to a password of your choice after you log in for the first time. Account Creators do not have access to
+   your password, and you should never share your password with anyone.
+3. **Requesters are required to adhere to the [Wikimedia Cloud Services End User Terms of Use][wmcs-tou-users].**
+
+### From Account Creators
+In order to use the Tool, we collect the following information from prospective Account Creators:
+
+- A username used to identify yourself on the Tool. This need not be your real name. It also need not be the same as
+  your Wikipedia username, however, we do also collect your Wikipedia username and associate it with your Tool username
+  (see below).
+- A password used to log in to the Tool. To protect your information, your password is salted and hashed using the
+  `bcrypt` algorithm before being stored in the Tool's database. **Please use a unique password. Do not use a password
+  that you use on any other site, and especially do not use the same password as used on any other Wikimedia project.**
+- An email address used to contact you about your account, to send you notifications about your account, and to reset
+  your password if you forget it.
+- Your Wikipedia username. We require you to have a Wikipedia account in good standing in order to use the Tool, and
+  you will need to allow the Tool to use OAuth to connect your account to verify your on-wiki identity. We also use
+  OAuth to allow the Tool to automatically create accounts on your behalf from within the Tool, instead of requiring you
+  to manually create them on-wiki. We will also check that your Wikipedia username appears on the list of users who have
+  signed the [Wikimedia Foundation Confidentiality Agreement for Nonpublic Information][wmf-confidentiality-agreement].
+
```

Comment:
IP and UA is also collected as part of the apache logs, but only accessible to tool roots and only persisted for 14 days.